src/test/java/com/appian/intellij/k/CompletionTest.java

```diff
@@ -17,9 +17,9 @@ protected String getTestDataPath() {
   }
 
   public void testGlobals() {
-    testCompletion(new String[] {"globals1.k", "globals2.k"}, "square", "sum",
+    testCompletion(new String[] {"globals1.k", "globals2.k"},
         "someFnInDefaultNsInAnotherFile", "save", "scan", "scov", "sdev", "select", "set", "setenv", "show",
-        "signum", "sin", "sqrt", "ss", "ssr", "string", "sublist", "sum", "sums", "sv", "svar", "system");
+        "signum", "sin", "sqrt", "square", "ss", "ssr", "string", "sublist", "sum", "sums", "sv", "svar", "system");
```

Comment:
"sum" function appeared twice in the expected results of the unit test, thus I did not need to create a new test, simply removed duplicate.